resources/views/largo/show.blade.php

```diff
@@ -6,6 +6,7 @@
     <script type="module">
         biigle.$declare('largo.user', {!! $user !!});
         biigle.$declare('largo.volumeId', {!! $volume->id !!});
+        biigle.$declare('largo.projectIds', {!! $volume->projects()->pluck('id') !!});
```

Comment:
Similar to above but here you can just do:

```suggestion
        biigle.$declare('largo.projectIds', {!! $projects->pluck('id') !!});
```